modules/default/compliments/compliments.js

```diff
@@ -1,4 +1,5 @@
 /* global Cron */
+/* global global */
```

Comment:
Why did you add this line? 'global' is already defined as a built-in global variable.

Reply:
was for trying to make earlier test global.window.name == 'jsdom' work... will remove
(as global was undefined)